webapp/src/Components/Player/Overview/PlayStyle/PlayerPlayStyleCard.tsx

```diff
@@ -20,7 +21,9 @@ export class PlayerPlayStyleCard extends React.PureComponent<Props> {
 
         return (
             <Card>
-                <CardHeader title={playStyleTitle} action={<PlayStyleActions player={this.props.player}/>}/>
+                <CardHeader title={playStyleTitle} action={<PlayStyleActions
```

Comment:
put action on the next line - the current whitespacing makes "player" and "handleplaylistchange" look like props of CardHeader.